info.json

```diff
@@ -28,17 +28,15 @@
         }
     ],
     "prerequisite": "* Ensure that SOC Simulator connector is configured\n* Ensure that MITRE ATT&CK Connector as well as data ingestion is configured",
-    "category": [],
     "certified": "true",
     "publisher": "Fortinet",
     "description": "The solution pack demonstrates users a variety of scenarios and use cases around threat hunting based on using the information provided by the MITRE ATT&CK Framework.",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-mitre-attack-threat-hunting/blob/develop/README.md",
```

Comment:
Description should contain connector name as suggested by Amit

Reply:
Added connector name in SP description